src/database/__init__.py

```diff
@@ -0,0 +1 @@
+from database.db_manager import DatabaseManager
```

Comment:
Because here we specified the whole relative position

Reply:
Ok so we talked about it and in fact, the __init__.py is called by itself when importing database. That's why in the previous example, the DatabaseManager class can be imported.